### ISSUE TITLE
No currency symbol in my country

### ISSUE THREAD

**Issue by Kimsoer** [roman910dev/currency_formatter#10]
Dear team currency_formatter!
Thank for your cool library

Can you add my currency symbol for my country?
khmer riel and code word : khr and symbol (៛)

https://en.wikipedia.org/wiki/Cambodian_riel

**Comment by roman910dev** [comment]
Hey! Thanks for your feedback!

The idea is to only include some of the most used currencies. There are too many currencies used worldwide and it can be very difficult to draw a line between those to include and those not to.

However, you can define your own custom currency and use it as any other. As a matter of fact, I have just published a new version (`2.2.0`), which tries to make the usage of custom currencies easier.

I have included a section on the best way of using custom currencies in the README which, by the way, uses your case as an example, because you were the one that motivated me to do this.

If you have any other concern regarding this topic, or if something does not fully suit you, I will be glad to further discuss it.

I copy here the part of the README I mentioned before.

## Using custom currencies

Let's say you want to use the following custom currency seamlessly:

```dart
CurrencyFormat khr = CurrencyFormat(
  symbol: '៛',
  symbolSide: SymbolSide.right,
);
```

You can create your own majors list:

```dart
const List<CurrencyFormat> myCurrencies = [
  ...CurrencyFormatter.majorsList,
  khr,
];
```

And use it to get the currency corresponding to a symbol or code, or the local one:

```dart
CurrencyFormat.fromSymbol('៛', myCurrencies); // khr
CurrencyFormat.fromCode('khr', myCurrencies); // khr

// custom local currency
CurrencyFormat? localCurrency() =>
    CurrencyFormat.fromSymbol(CurrencyFormat.localSymbol, myCurrencies);
// or
CurrencyFormat? localCurrency() =>
    CurrencyFormat.fromLocale(null, myCurrencies);
```

**Comment by Kimsoer** [comment]
Okay thank you so much @roman910dev
